### PR TITLE
Schiphol Airport (Independent Publisher)

### PR DESCRIPTION
**Pull request by Miesjh**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 
image
image
image
image

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Congratulations on your first Independent Publisher connector @Miesjh! Thank you for your contribution :)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@Miesjh Please resolve all the errors mentioned below, and please make sure that you are validating your connector using our paconn tool before pushing commit to the PR.

https://github.com/microsoft/PowerPlatformConnectors/tree/dev/tools/paconn-cli#validate-a-swagger-json

##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1flights/get/parameters/15
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1flights/get/parameters/16
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1flights/get/parameters/17
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1flights/get/parameters/18
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1airlines/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1airlines/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1airlines/get/parameters/2
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1airlines/get/parameters/3
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1airlines/get/parameters/4
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1airlines/get/parameters/5
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1airlines~1{airline}/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1airlines~1{airline}/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1airlines~1{airline}/get/parameters/2
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1airlines~1{airline}/get/parameters/3
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1airlines~1{airline}/get/parameters/4
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1aircrafttypes/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1aircrafttypes/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1aircrafttypes/get/parameters/2
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1aircrafttypes/get/parameters/3
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1aircrafttypes/get/parameters/4
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1aircrafttypes/get/parameters/5
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1aircrafttypes/get/parameters/6
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1aircrafttypes/get/parameters/7
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1destinations/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1destinations/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1destinations/get/parameters/2
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1destinations/get/parameters/3
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1destinations/get/parameters/4
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1destinations/get/parameters/5
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1destinations~1{iata}/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1destinations~1{iata}/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1destinations~1{iata}/get/parameters/2
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1destinations~1{iata}/get/parameters/3
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1destinations~1{iata}/get/parameters/4
##[error]The 'x-ms-connector-metadata' property is required.
##[warning]independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json(,): warning : Encoding for path parameter 'id' is missing, which might lead to invalid requests. Use "x-ms-url-encoding": "single" to ensure the value is URL encoded. ValuePath: /paths/~1flights~1{id}/get/parameters/4/x-ms-url-encoding
##[warning]independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json(,): warning : Encoding for path parameter 'airline' is missing, which might lead to invalid requests. Use "x-ms-url-encoding": "single" to ensure the value is URL encoded. ValuePath: /paths/~1airlines~1{airline}/get/parameters/4/x-ms-url-encoding
##[warning]independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json(,): warning : Encoding for path parameter 'iata' is missing, which might lead to invalid requests. Use "x-ms-url-encoding": "single" to ensure the value is URL encoded. ValuePath: /paths/~1destinations~1{iata}/get/parameters/4/x-ms-url-encoding

File: independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,869 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Schiphol Airport",
```
[the remark this answers is not in the document]
Comment (Miesjh) [reply independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json]: Fixed

File: independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,869 @@
...
+    "version": "4.1",
```
[the remark this answers is not in the document]
Comment (Miesjh) [reply independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json]: Fixed

File: independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,869 @@
...
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/Flight" }
+          }
+        }
+      }
+    },
+    "/flights": {
+      "get": {
+        "tags": [ "flight-controller" ],
+        "summary": "Flights",
```
[the remark this answers is not in the document]
Comment (Miesjh) [reply independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json]: Fixed

File: independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,869 @@
...
+        "description": "Retrieves flights for a specific date. If scheduleDate is provided as query parameter, it will retrieve flights for that date. If not provided it is defaulted to today. If there are more hits then allowed on one page response headers will contain the URLs where other pages of the results can be reached.For example:\u003cbr/\u003eLink: \u0026lt;protocol://server_address:port/public-flights/resource\u0026gt; ; rel=\"first\", \u003cbr/\u003e\u0026lt;protocol://server_address:port/public-flights/resource?page=10\u0026gt; rel=\"last\", \u003cbr/\u003e\u0026lt;protocol://server_address:port/public-flights/resource?page=3\u0026gt; rel=\"next\", \u003cbr/\u003e\u0026lt;protocol://server_address:port/public-flights/resource?page=1\u0026gt; rel=\"prev\"\u003cbr/\u003e\u003cbr/\u003e",
```
[the remark this answers is not in the document]
Comment (Miesjh) [reply independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json]: Fixed

File: independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,869 @@
...
+      "name": "Michel Gueli",
+      "url": "https://www.teqtive.nl",
+      "email": "michel@teqtive.nl"
+    }
+  },
+  "host": "api.schiphol.nl",
+  "basePath": "/public-flights",
+  "schemes": [ "https" ],
+  "tags": [
+    { "name": "flight-controller" },
```
[the remark this answers is not in the document]
Comment (Miesjh) [reply independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json:17]: I do not understand what you mean here. Visual Studio is formatting is it for me.

[the remark this answers is not in the document]
Comment (Miesjh) [reply independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json]: Fixed. What's wrong in the apiProperties.json File?

File: independent-publisher-connectors/Schiphol Airport/Readme.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+## Prerequisites
+You need a Schiphol Developer account. You can get one for free [here](https://developer.schiphol.nl/signup).
+
+## Obtaining Credentials
+In all requests both the APP ID and APP KEY have to be provided. These settings are visible in the Dashboard, Show API Keys.
+
+## Supported Operations
+
+### Retrieve Aircraft Types
+Retrieves list of aircrafttypes. [More info](https://developer.schiphol.nl/apis/flight-api/v4/flights?version=latest#!/aircraft-type-controller/retrieveAllAircraftTypesUsingGET)
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Schiphol Airport/Readme.md:17]: @Miesjh there's a typo in `aircrafttypes`. Please make sure everything is grammatically correct and are free of typographical errors. You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,1187 @@
...
+      "name": "aircraft-type-controller" 
+
+    },
+    {
+      "name": "destination-controller" 
+
+    },
+    {
+      "name": "airline-controller" 
+
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json:33]: @Miesjh please remove empty newlines.

File: independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,1187 @@
...
+      }
+    },
+    "/flights": {
+      "get": {
+        "tags": [
+          "flight-controller"
+        ],
+        "summary": "Retrieve Flights",
+        "x-ms-summary": "Retrieve Flights",
+        "description": "Retrieves flights for a specific date. If scheduleDate is provided as query parameter, it will retrieve flights for that date. If not provided it is defaulted to today. If there are more hits then allowed on one page response headers will contain the URLs where other pages of the results can be reached.",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json:119]: @Miesjh please make this description a bit shorter and more precise.

File: independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json [hunk 9]
```diff
@@ -0,0 +1,1187 @@
...
+            "description": "NVLS code of an airliner.",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "x-ms-summary": "Airline Code"
+          },
+          {
+            "name": "route",
+            "in": "query",
+            "description": "IATA or ICAO code of airport in route; multiple values, comma separated",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json:219]: @Miesjh please [add period](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#summary-and-description) at the end of description.

File: independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json [hunk 10]
```diff
@@ -0,0 +1,1187 @@
...
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RowType": {
+      "properties": {
+        "Position": {
+          "type": "string",
+          "x-ms-summary": "Belts"
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json:724]: @Miesjh you have two same `x-ms-summary` with `Belts`, please check if it's correct.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @Miesjh! Please take a look at the comments, thank you!

File: independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json (hunk 10 above)
Comment (Miesjh) [reply independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json]: Fixed

**Comment by v-EgorKozhadei** [comment]
Hi @Miesjh! Can you please fix the minor changes from the latest comments? Thanks in advance!

**Comment by Miesjh** [comment]
> Hi @Miesjh! Can you please fix the minor changes from the latest comments? Thanks in advance!

I did. What are you missing?

**Comment by v-EgorKozhadei** [comment]
Hi @Miesjh! Please check new comments and resolve once fixes are done.

> > Hi @Miesjh! Can you please fix the minor changes from the latest comments? Thanks in advance!
> 
> I did. What are you missing?

File: independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json [hunk 11]
```diff
@@ -0,0 +1,1183 @@
...
+    },
+    "/aircrafttypes": {
+      "get": {
+        "tags": [
+          "aircraft-type-controller" 
+        ],
+        "summary": "Retrieve Aircraft Types",
+        "x-ms-summary": "Retrieve Aircraft Types",
+        "x-ms-visibility": "important",
+        "description": "Retrieves list of aircrafttypes. If there are more hits then allowed on one page response headers will contain the URLs where other pages of the results can be reached.",
```
[the remark this answers is not in the document]
Comment (Miesjh) [reply independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json]: Fixed

File: independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json [hunk 12]
```diff
@@ -0,0 +1,1183 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Schiphols flight information offers a wealth of information on current and scheduled flights to and from the airport.",
```
[the remark this answers is not in the document]
Comment (Miesjh) [reply independent-publisher-connectors/Schiphol Airport/apiDefinition.swagger.json]: Fixed

**Comment by sriyen-msft** [comment]
@Miesjh We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. Thanks.

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @Miesjh,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by sriyen-msft** [comment]
@Miesjh Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**